components/buttons/buttons.css

```diff
@@ -0,0 +1,150 @@
+/* BUTTON COMPONENT START */
+
+/* Contained Button */
+/* Contained Defualt Button */
+.btn{
+    display: inline-block;
+    vertical-align: middle;
+    width: max-content;
+    min-width: 100px;
+    height: max-content;
+
+    font-weight: bold;
+    background: var(--default-color);
+    color: var(--off-white);
+    
+    padding: 0.8rem 1rem;
+    border-radius: 0.5rem;
+    border: none;
```

Comment:
Instead of giving border as none you can keep it transparent because it might cause issues when trying to apply styles on special cases such as hover or click